gulp/tasks/copy.js

```diff
@@ -75,6 +83,7 @@ gulp.task( 'copy',
   gulp.parallel(
     'copy:icons',
     'copy:lightbox2',
+    'copy:erap',
```

Comment:
Why's the copy task necessary? Can we put `cfgov/unprocessed/erap/js/main.js` in `cfgov/unprocessed/js/routes/on-demand/erap/main.js` to start with?

Where is the source for this main.js file?

Reply:
@anselmbradford The tool is maintained as an [external standalone React app](https://github.com/cfpb/rental-assistance-finder). At the moment we're compiling and importing the JS directly because the app hasn't yet been released as an npm module. Once it is, the copy task will move the built JS file from `node_modules` to its `static_built` destination, like the existing lightbox2 task, and the current `unprocessed` source directory will be removed.

Reply:
At the moment we're planning to have the tool on two pages, the English page that's launching today and a Spanish page (with translated tool content) that will be added in the near future.